### DISCUSSION TITLE
我是否能不遵守配置式路由，自己使用组件进行路由？

### DISCUSSION THREAD

**Discussion by YqxLzx** [umijs/umi#10543] [Q&A]
say no : 
```
[
    {
      path:'/admin/appList',
      component: '@/pages/Admin/children/appController',
  }
]
```
say yes :
`<BrowserRouter basename="/my-app">
      <Routes>
        <Route path="/" element={<Home />} />
      </Routes>
   </BrowserRouter>`

**Comment by fz6m** [comment]
路由部分是 umi 框架内的约定，如需自定义路由，请选择 vite 等从 0 开始的框架，以便自定义路由。
